mail.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"mime/multipart"
+	"net/mail"
+	"net/smtp"
+	"net/textproto"
+	"os"
+	"strings"
+	"time"
+
+	b64 "encoding/base64"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	mailSubject      = "MACFin portal test accounts"
+	AttachedFileName = "MACFin_test_accounts.xlsx"
+	bodyEncoding     = "base64"
+	charSet          = "UTF-8"
+)
+
+const (
+	FileMimeType           = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
+	FileContentDisposition = "attachment"
+)
+
+const (
+	SMTPHost = "internal-Enterpris-SMTPProd-I20YLD1GTM6L-357506541.us-east-1.elb.amazonaws.com"
+	SMTPPort = "25"
+)
+
+type SMTPRelay struct {
+	Host string
+	Port string
+}
+
+type Message struct {
```

Comment:
Are you planning to move this `Message` and `File` API into a package and use it elsewhere? Otherwise I'm having trouble understanding the advantage of creating these APIs just to use them once in the same file. The disadvantage is that now in addition to making sure each of these higher-level APIs uses the low-level APIs correctly, we need to make sure that the higher-level APIs are themselves following their contracts, and that `sendEmail` is using them correctly.

What do you think about, instead, moving most of this logic to `sendEmail`, with maybe occasional helper functions if you think some of the logic doesn't need to be inline. For example, you could write:

```
func validateAndDeduplicateRecipientAddresses(addresses []string) ([]string, err)
```

Reply:
Thank you for the explanation. I have removed the Message and File structs and methods and I moved the logic into `sendEmail`. I deployed the code and verified it works as before.